administrator/components/com_joomlaupdate/extract.php

```diff
@@ -0,0 +1,2046 @@
+<?php
+/**
+ * @package         Joomla.Administrator
+ * @subpackage      com_joomlaupdate
+ *
+ * @copyright       (C) 2021 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license         GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+// Uncomment to enable debugging and prevent extract.php from being overwritten during the update
+//define('_JOOMLA_UPDATE_DEBUG', 1);
```

Comment:
```suggestion
// define('_JOOMLA_UPDATE_DEBUG', 1);
```
Make the bot happy again.

Reply:
OK, this is ridiculous. It doesn't like it with or without the space.

At this point I am going to replace that with stupid instructions and may God have mercy on the poor soul who has to debug this file.

Reply:
It was ok with the space but that space got lost in the last commit. Possibly PHPCS will complain about multiline comment, in this case ust add an empty line between the comment and that commented out line.

Reply:
The reason why the space got lost was maybe because it was an applied suggestion from here, while later commits have been pushed up.

Reply:
I was getting a problem with the multiline comment being two consecutive line comments. Since we can't tell the bot to shut up and let us provide a better troubleshooting experience I replaced those two lines with more verbose instructions which, in retrospect, may be easier if someone is trying to debug that file and I am not around to tell them how.

Making the bot happy and improving the bus factor? Okay, I'll take it.